src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -213,9 +247,16 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
 
   ngOnInit() {
     this.wrappedInjector = new HostWrapper(WrappedRow, this.vcr);
+    this.oldHeight = this.getCurrentHeight();
+    this.expandedState = this.expand.expanded ? 'expanded' : 'collapsed';
   }
 
   public get _view() {
     return this.wrappedInjector.get(WrappedRow, this.vcr).rowView;
   }
+
+  private getCurrentHeight() {
+    const height = this.domAdapter.computedHeight(this.el.nativeElement) || 0;
+    return height + 'px';
```

Comment:
nit: can we just return the number without turning it into a string with the `px`. There are some cases where we need to use just a plain number value of dom properties. If we need to turn it into px, we can do so by`${this.getCurrentHeight()}px`. Once it's returning px value in a string, we have to parse it back in order to get the number version.

Reply:
done